<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrollViewPropertyGridWidget</class>
 <widget class="QWidget" name="ScrollViewPropertyGridWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>192</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>192</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>191</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDoubleSpinBox" name="scrollHPositionSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>109</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="scrollHSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSlider" name="scrollVSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>162</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scrollVPositionSpinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>159</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="verticalScrollLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical scroll position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="horizontalScrollLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal scroll position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scrollAreaSizeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Scroll Area Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeXLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeYLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size Y:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="contentSizeXSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>55</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="contentSizeYSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>55</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
